packages/gdl-frontend/pages/books/translations.js

```diff
@@ -333,9 +335,126 @@ const CustomGrid = styled('div')`
   }
 `;
 
-class MyTranslationsPage extends React.Component<{
-  intl: intlShape
-}> {
+const CustomSearchBar = styled('div')`
+  display: flex;
+  flex-direction: column;
+  @media (min-width: ${TABLET_BREAKPOINT}px) {
+    flex-direction: row;
+    justify-content: space-between;
+  }
+`;
+
+const CustomSearchbarItem = styled('div')`
+  width: 100%;
+  padding-top: 5px;
+  @media (min-width: ${TABLET_BREAKPOINT}px) {
+    padding-top: 0;
+    width: 30%;
+  }
+`;
+
+function dynamicSort(property) {
```

Comment:
Her har du en funksjon som gjør akkurat det samme 3 ganger med input `a` og `b`.

Hva hvis du lager en funksjon som tar inn to inputparametre også returnerer det den skal, så kan du kalle den tre ganger isteden for å skrive nesten lik kode 3 ganger :) Feks:

```javascript
function compareValues(a, b) {
return a < b ? -1 : a > b ? 1: 0;
}
// Brukes slik:
const result = compareValues(a.to.language['name], b.to.language['name'])
```

Du kan også sende med sortOrder :)